README.md

```diff
@@ -500,8 +500,11 @@ can use most commands you know from the emacs package menu. This
 option requires the rust package `cargo-outdated` to be installed
 before being used.
 
-- `u` mark single crate for upgrade
-- `U` mark all upgradable crates
+- `u` mark single crate for upgrade. Asks using minibuffer the exact
+  version that should be used for upgrade.
```

Comment:
"mark single crate for upgrade. Asks using minibuffer the exact
  version that should be used for upgrade"
-> 
"mark single crate for upgrade and prompt user for version"

Reply:
Fixed.